scripts/tools/ManageAssociationMap.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoTaskQueue\scripts\tools;
+
+use InvalidArgumentException;
+use oat\oatbox\action\Action;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\tao\model\taskQueue\Queue;
+use oat\tao\model\taskQueue\QueueDispatcher;
+use oat\tao\model\taskQueue\QueueDispatcherInterface;
+use oat\taoTaskQueue\model\QueueBroker\RdsQueueBroker;
+use Zend\ServiceManager\ServiceLocatorAwareTrait;
+
+class ManageAssociationMap extends ScriptAction
+{
+    use ServiceLocatorAwareTrait;
+
+    protected function provideOptions()
+    {
+        return [
+            'queue' => [
+                'prefix' => 'q',
+                'longPrefix' => 'queue',
+                'required' => false,
+                'cast' => 'string',
+                'description' => 'Define task queue to add'
+            ],
+
+            'taskClass' => [
+                'prefix' => 't',
+                'longPrefix' => 'taskClass',
+                'required' => true,
+                'cast' => 'string',
+                'description' => sprintf(
+                    'Define task (must extend "%s")',
+                    Action::class
+                )
+            ],
+        ];
+    }
+
+    protected function provideDescription()
+    {
+        // TODO: Implement provideDescription() method.
+    }
+
+    protected function provideUsage()
+    {
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Prints the help.'
+        ];
+    }
+
+    protected function run()
```

Comment:
I did not test / used it before:

1) I think we could at least extract the logic from `run()` to methods (or classes) with meaningful names. 
2) Also as its a new important feature, should be add to the README.md how/why to use it.

Reply:
https://github.com/oat-sa/extension-tao-task-queue/pull/117/commits/fbf7e0057f665af412898f69e0aeb14c4ccd7dcc